src/cdk/a11y/a11y.md

```diff
@@ -203,15 +205,38 @@ class:
 
 #### Targeting high contrast users
 
-The `a11y` package offers a mixin that allows you to target users that have the Windows high
-contrast mode turned on. To target high contrast users, you can wrap your styles with the
-`cdk-high-contrast` mixin. The mixin works by targeting a CSS class which is added to the `body`
-by the CDK when high contrast mode is detected at runtime.
+Microsoft Windows includes an accessibility feature called [Windows High Contrast Mode][]. This
+mode is supported by Microsoft Edge and Firefox. Google Chrome does not support high contrast mode.
+
+The `cdk/a11y` package provides a Sass mixin that lets you define styles that only apply in high
+contrast mode. To create a high contrast style, define your style inside the `high-contrast` mixin.
+
+The mixin works by targeting a CSS class which is added to the `body` by the CDK when high contrast
+mode is detected at runtime, via the `HighContrastModeDetector` service.
 
 ```scss
+@use '~@angular/cdk';
+
 button {
-  @include cdk-high-contrast {
+  @include cdk.high-contrast() {
     outline: solid 1px;
   }
 }
 ```
+
+The `high-contrast` mixin accepts two optional parameters, `$target` and `$encapsulation`.
+
+The `$target` parameter allows you to specify which variation of high contrast mode your style
+targets. The accepted values are `active` (default), `black-on-white`, and `white-on-black`. These
+values correspond to the supported values for the
+[`-ms-high-contrast` media query][ms-high-contrast].
```

Comment:
Not sure whether we should mention the media query, because we wrap the selector with a class which behaves slightly differently when used together with Sass' `&`.

Reply:
I still want to mention it because it explains where the values come from

Reply:
Sure, I don't feel strongly against it.